server/src/main/java/org/elasticsearch/action/admin/indices/alias/get/TransportGetAliasesAction.java

```diff
@@ -63,7 +76,61 @@ protected GetAliasesResponse newResponse() {
     protected void masterOperation(GetAliasesRequest request, ClusterState state, ActionListener<GetAliasesResponse> listener) {
         String[] concreteIndices = indexNameExpressionResolver.concreteIndexNames(state, request);
         ImmutableOpenMap<String, List<AliasMetaData>> result = state.metaData().findAliases(request.aliases(), concreteIndices);
-        listener.onResponse(new GetAliasesResponse(result));
+
+        String message = null;
+        RestStatus status = null;
```

Comment:
do we end up passing in `null` here unless it's NOT_FOUND ? Sorry about the confusion, I meant that it's ok to have null for cases where we don't really have the value (we didn't serialize it) but in this case we should rather provide the OK value. Makes sense?